moto/sqs/responses.py

```diff
@@ -202,8 +202,14 @@ def get_queue_attributes(self):
 
     def set_queue_attributes(self):
         # TODO validate self.get_param('QueueUrl')
+        attr_name = self.querystring.get('Attribute.1.Name')[0]
```

Comment:
Thanks for this PR @mgagliardo !

The request may have multiple attributes, so I think we should find a generic solution, rather than hardcoding `.1.` here. Can we create a small util function that extracts all attributes from `querystring`? 

Also, there is already a util function `_get_multi_param()` - I'm assuming this one does not get us the correct result?
```
attribute_names = self._get_multi_param("AttributeName")
```

Reply:
Thanks for your comment Waldemar! And yes that is correct, I have now fixed this over the matching `Attribute.N.Name` k/v and it works like a charm